client/src/util/stateManager/matrix.ts

```diff
@@ -12,30 +12,68 @@ import {
   KeyIndex,
 } from "../dataframe";
 
+import { CatInt8Array } from "./cat8_array";
+import { CatInt16Array } from "./cat16_array";
+import { CatInt32Array } from "./cat32_array";
+
 const utf8Decoder = new TextDecoder("utf-8");
 
 /**
  * Matrix flatbuffer decoding support. See fbs/matrix.fbs
  */
 
 /**
- * Decode NetEncoding.TypedArray
+ * Decode NetEncoding.TypedFBArray
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
```

Comment:
My understanding is that the team agreement is that these eslint disables will be fixed whenever major changes are made to a file.  I definitely think that they are hiding type errors, so it is worth cleaning them up IMHO.

Reply:
Decoder fixed, but encoder is giving me problems. Will keep trying.